apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -1442,8 +1443,66 @@ describe('LIVECHAT - rooms', function () {
 				.expect('Content-Type', 'application/json')
 				.expect(400);
 		});
+		(IS_EE ? it : it.skip)('should update room priority', async () => {
```

Comment:
Can't you just use `test.skip` at the beginning of the file, like we do in the FE?

Reply:
I don't think so, that seems to be something that playwright offers however couldn't find something similar on mocha

Reply:
Yeah, mocha is not that good. 
On the other hand, making it a global handler would be good: `eeDescribe` or `eeIt` (naming could be better obvs)

but that's something OOS for this task